logging/cluster-logging-release-notes.adoc

```diff
@@ -372,21 +386,25 @@ This release includes link:https://access.redhat.com/errata/RHSA-2021:5129[RHSA-
 
 [id="cluster-logging-release-notes-5-3-0_{context}"]
 == OpenShift Logging 5.3.0
-The following advisories are available for OpenShift Logging 5.3.x:
+This release includes link:https://access.redhat.com/errata/RHSA-2021:4627[RHSA-2021:4627 OpenShift Logging Bug Fix Release 5.3.0]
 
 [id="openshift-logging-5-3-0-new-features-and-enhancements_{context}"]
 === New features and enhancements
-* With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
+* With this update, authorization options for Log Forwarding have been expanded. Outputs may now be configured with SASL, username/password, or TLS.
 
 [id="openshift-logging-5-3-0-bug-fixes_{context}"]
 === Bug fixes
+* Before this update, if you forwarded logs using the syslog protocol, serializing a ruby hash encoded key/value pairs to contain a '=>' character and replaced tabs with "#11". This update fixes the issue so that log messages are correctly serialized as valid JSON. (link:https://issues.redhat.com/browse/LOG-1494[LOG-1494])
```

Comment:
The verb tenses and commas in this sentence make it confusing. Can you rephrase this, please?

Reply:
`Ruby hash-encoded key-value pairs`